index.html

```diff
@@ -5,232 +5,250 @@
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="img/Vector (10).png" alt="peace symbol">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="img/Vector (11).png" alt="twitter picture">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="img/Vector (12).png" alt="Gmail picture">
+                            </a>
+                        </li>
+                    </ul>
+                    <button class="btn-resume">Get my resume</button>
+                </div>
+                <div class="tool">
+                    <button class="language-button">Languages <span class="fa fa-chevron-down"></span></button>
+                    <ul class="languages">
+                        <li>
+                            <a href="#"><img src="img/Ellipse 1 (3).png" alt="not found"/><span>javascript</span></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="img/Ellipse 1 (4).png" alt="not found"/><span>html</span></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="img/Ellipse 1 (5).png" alt="not found"/><span>css</span></a>
+                        </li>
+                    </ul>
+                    <button>frameworks <span class="fa fa-chevron-right"></span></button>
+                    <button>Skills <span class="fa fa-chevron-right"></span></button>
+                </div>
             </div>
         </section>
-    </main>
+
+        <footer>
+            <section class="contact">
+                <div class="contact-container">
+
+                    <h2>Contact me</h2>
+                    <p>
+                        If you have an application you are interested in developing, a feature
+                        that you need built or a project that needs coding. I’d love to help
+                        with it.
+                    </p>
+                    <form action="https://formspree.io/f/mrgjrqoe" method="POST">
+                        <input type="test" name="name" placeholder="Full name" maxlength="30" required />
+                        <input type="email" name="email address" placeholder="Email address" required />
+                        <textarea name="contact" cols="30" rows="10" maxlength="500"
+                            placeholder="write you message here" required></textarea>
+                        <div class="btn">
+                            <button type="submit">Get in touch</button>
+                        </div>
+                    </form>
+                </div>
+            </section>
+        </footer>
+    </section>
```

Comment:
- [ ] Kindly note that I'm suggesting this changes because it's a new introduced bug. Your about page section seem to to now have some spaces by the side which isn't in the original design.

- Your design.
<img width="467" alt="image" src="https://github.com/Dorcas126/Setup-and-mobile-first-project/assets/41163571/c7724839-ccb0-4368-aded-924c0ad1130f">

- Figma design
<img width="295" alt="image" src="https://github.com/Dorcas126/Setup-and-mobile-first-project/assets/41163571/10f03775-20bf-40fc-8ada-98f792291237">